pulsar-broker/src/main/java/org/apache/pulsar/broker/stats/BrokerOperabilityMetrics.java

```diff
@@ -38,6 +38,7 @@ public class BrokerOperabilityMetrics {
     private final LongAdder connectionCreateFailCount;
     private final LongAdder connectionTotalClosedCount;
     private final LongAdder connectionActive;
+    private volatile Long healthCheckStatus;
```

Comment:
```
default value -1 means unknown
1 = success
0 = fail
```
so, if no one enables this task then value should be -1 unknown.

Reply:
addressed